server/exercise/src/main/scala/com/eigengo/lift/exercise/UserExercisesProcessor.scala

```diff
@@ -224,10 +225,18 @@ class UserExercisesProcessor(notification: ActorRef, userProfile: ActorRef)
         .fold({ err ⇒ persist(SinglePacketDecodingFailedEvt(id, err, bits)) { evt ⇒ sender() ! \/.left(err) } },
               { dec ⇒ persist(ClassifyExerciseEvt(sessionProps, dec)) { evt ⇒ classifier ! evt; sender() ! \/.right(()) } })
 
+    //TODO: Handle all and not just first
     case ExerciseDataProcessMultiPacket(`id`, packet) ⇒
-      persist(MultiPacketDecodingFailedEvt(id, "Not implemented", packet)) {
-        evt ⇒ sender() ! \/.left("Not implemented")
-      }
+      log.debug("ExerciseDataProcess: exercising -> exercising.")
+
+      val firstSensorData = packet.rawSensorData.head.data
+      tracing ! firstSensorData
```

Comment:
The `tracing` ActorRef gets the first decoded packet, but does it get the other decoded packets? If so, it might help to add in a comment regarding this?